app/views/layouts/admin.html.haml

```diff
@@ -11,7 +11,7 @@
   %body{class: body_class(request)}
     = render "shared/flash"
     %div.wrapper
-      = render "shared/nav"
+      #nav_root{"data-rooturl" => main_app.root_url, "data-logopath" => logo_path, "data-exploreclass" => 'inactive', "data-fluid" => "false", "data-exploreurl" =>  main_app.explore_path, "data-explorename" => t(Features.wiki_ed? ? "application.explore" : "courses_generic.explore"), "data-usersignedin" => user_signed_in? ? 'true' : 'false', "data-ifadmin" => current_user&.admin? ? 'true' :'false', "data-trainingurl": main_app.training_url, "data-disable_training": Features.disable_training? ? 'true' : 'false', "data-help_disabled" => Features.disable_help? ? 'true' : 'false', "data-ask_url" => main_app.ask_url, "data-user_permissions" => permissions, "data-wiki_ed" => Features.wiki_ed? ? 'true' : 'false', "data-language_switcher_enabled" => languageswitcherenabled, "data-username" => current_user&.username, "data-destroyurl" => main_app.destroy_user_session_url, "data-omniauth_url" => main_app.user_mediawiki_omniauth_authorize_url}
```

Comment:
The main.<hash>.js file is not loaded in the admin layout. You'll need to add it. That may be a problem with other layouts as well.